hyrisecockpit/workload_generator/workloads/job_1/6a.sql

```diff
@@ -1,5 +1,5 @@
-SELECT MIN(k.keyword) AS movie_keyword, MIN(n.name) AS actor_name, MIN(t.title) AS marvel_movie
-FROM cast_info AS ci, keyword AS k, movie_keyword AS mk, name AS n, title AS t
+SELECT MIN(k.keyword) AS movie_keyword_job_1, MIN(n.name) AS actor_name, MIN(t.title) AS marvel_movie
```

Comment:
This should not be renamed:
```suggestion
SELECT MIN(k.keyword) AS movie_keyword, MIN(n.name) AS actor_name, MIN(t.title) AS marvel_movie
```